tutorials/plot_spatial.py

```diff
@@ -11,40 +11,215 @@
 # Apply spatial measures to spatial data
 # --------------------------------------
 #
-# Words, words, words.
+# This tutorial contains the following sections:
+#
+# 1. Compute and plot distances and speed using position
+# 2. Divide position in spatial bin edges
+# 3. Compute spatial bin assignment using spatial bin edges
+# 4. Compute time in each timestamp sample
+# 5. Compute and plot occupancy
+# 6. Compute 2D and 1D spatial information
 #
 
 ###################################################################################################
 
 # sphinx_gallery_thumbnail_number = 1
 
-import spiketools
+# import auxiliary libraries
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+import all functions from spiketools.spatial
+from spiketools.spatial.position import compute_distance, compute_distances, compute_cumulative_distances, compute_speed
+from spiketools.spatial.occupancy import compute_spatial_bin_edges, compute_spatial_bin_assignment, compute_bin_time, compute_occupancy
```

Comment:
In general, we try to keep line length below 100 characters (which is the length of the `####` lines). 
For multi-line imports, this can be updated to look something like:
```
 from spiketools.spatial.occupancy import (compute_spatial_bin_edges, compute_spatial_bin_assignment,
                                           compute_bin_time, compute_occupancy)
```